roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -13,54 +13,46 @@
     group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   loop:
-    - {"path": "/var/lib/openstack/config/libvirt"}
+    - {"path": "/etc/libvirt", "owner": "root", "group": "root", "mode": "0755"}
```

Comment:
why do we need to change the rpm shipped permissions?

Reply:
we are creating it before the rpm is installed
but also the rpm does not create a libvirt user it runs libvirt as root
so this is the same permission it would have anyway

Reply:
Let's ensure RPM installed before doing that?
Let's run it as root? My concern is libvirt team could tell us later they don't test running it as non-root, like they don't for running it w/o systemd containerd integrated cgroups management. And we will on our own support again

Reply:
ya we talked about this else where.
im going to reoder the install to happne before configure and defer starting the servics until after.
that should also resolve the selinux isues as the folder will be created with the correct se linux lables.